sources/packages/backend/libs/services/src/disbursement-overaward/disbursement-overaward.service.ts

```diff
@@ -70,4 +75,58 @@ export class DisbursementOverawardService {
     }
     return result;
   }
+
+  /**
+   * Get all overawards which belong to a student.
+   * @param studentId student.
+   * @returns overaward details of a student.
+   */
+  async getOverawardsByStudent(
+    studentId: number,
+  ): Promise<DisbursementOveraward[]> {
+    return this.disbursementOverawardRepo.find({
+      select: {
+        createdAt: true,
+        originType: true,
+        overawardValue: true,
+        disbursementValueCode: true,
+        creator: { firstName: true, lastName: true },
+        studentAssessment: {
+          id: true,
+          application: { applicationNumber: true },
+          triggerType: true,
+        },
+      },
+      where: {
+        student: { id: studentId },
+      },
+      relations: {
```

Comment:
I would say that we have been configuring the object query following the order as `select`, `relations`, and `where`.